Packs/Cymulate/Integrations/Cymulate_v2/Cymulate_v2.yml

```diff
@@ -578,54 +452,40 @@ script:
       description: Template name.
       type: String
   - arguments:
-    - default: false
-      description: Agent name to run the assessment with.
-      isArray: false
+    - description: Agent name to run the assessment with.
       name: agent_name
       required: true
-      secret: false
-    - default: false
-      description: The ID of the template to run the lateral movement with. Can be retrieved using Cymulate's UI, or using cymulate-lateral-movement-template-list command.
-      isArray: false
+    - description: Agent profile name to run the assessment with.
+      name: agent_profile_name
+      required: true
```

Comment:
Is this on purpose (`required: true`), it might break compatibility for customers who are already running the command, and suddenly now they have to add another argument

Reply:
yes they are required

Reply:
I have pushed the changes, will it be automatically updated in current pr or do I need to do new PR?